kafka_connect_exporter.go

```diff
@@ -136,13 +136,19 @@ func (e *Exporter) Collect(ch chan<- prometheus.Metric) {
 
 		for _, connectorTask := range connectorStatus.Tasks {
 
-			var isTaskRunning float64 = 0
+			var taskState float64 = 0
 			if strings.ToLower(connectorTask.State) == "running" {
-				isTaskRunning = 1
+				taskState = 1
+			}
+			if strings.ToLower(connectorTask.State) == "unassigned" {
+				taskState = 2
+			}
+			if strings.ToLower(connectorTask.State) == "paused" {
+				taskState = 3
 			}
 
 			ch <- prometheus.MustNewConstMetric(
-				areConnectorTasksRunning, prometheus.GaugeValue, isTaskRunning,
+				areConnectorTasksRunning, prometheus.GaugeValue, taskState,
```

Comment:
can you also update the `HELP` section for the [metric](https://github.com/wakeful/kafka_connect_exporter/blob/master/kafka_connect_exporter.go#L36) (and Readme file)?

Reply:
updated. please take a look